### PR TITLE
fix(akita-ng-effects): fix registration of multiple for features

### PR DESCRIPTION
**Pull request by yackinn**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/datorama/akita/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: libs/akita-ng-effects/src/lib/akita-ng-effects.module.ts [hunk 1]
```diff
@@ -35,6 +35,7 @@ export class AkitaNgEffectsModule {
         {
           provide: _FEATURE_EFFECTS,
           useValue: featureEffects,
+          multi: true,
```
Comment (NetanelBasal) [inline libs/akita-ng-effects/src/lib/akita-ng-effects.module.ts:38]: Did you check this? `multi` will change it to be an array, so it should be a multi-dimensional array now.

Comment (yackinn) [reply libs/akita-ng-effects/src/lib/akita-ng-effects.module.ts:38]: Let me check again.

Comment (yackinn) [reply libs/akita-ng-effects/src/lib/akita-ng-effects.module.ts:38]: It is. You're correct. I changed it.

**Comment by yackinn** [comment]
Wait before accepting. I need to check more.

**Comment by yackinn** [comment]
I added more for feature effects. It's working as intended now.

File: libs/akita-ng-effects/src/lib/module-manager.service.ts [hunk 2]
```diff
@@ -22,23 +21,27 @@ export class ModuleManager implements OnDestroy {
   subscribeToEffects(effectInstance: Type<any>) {
     for (let key in effectInstance) {
       const property: Effect = effectInstance[key];
+      // console.log("property", property)
```
Comment (NetanelBasal) [inline libs/akita-ng-effects/src/lib/module-manager.service.ts:24]: Remove please

File: apps/angular-store-app/src/app/posts/state/post.effects.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { Injectable } from '@angular/core';
+import { Actions, Effect } from '@datorama/akita-ng-effects';
+import { PostsStore } from './posts.store';
+import { tap } from 'rxjs/operators';
```
Comment (NetanelBasal) [inline apps/angular-store-app/src/app/posts/state/post.effects.ts:4]: Both modules are lazy-loaded. I'd test it on eager modules.

File: libs/akita-ng-effects/src/lib/module-manager.service.ts (hunk 2 above)
Comment (yackinn) [reply libs/akita-ng-effects/src/lib/module-manager.service.ts]: removed

File: apps/angular-store-app/src/app/posts/state/post.effects.ts (hunk 3 above)
Comment (yackinn) [reply apps/angular-store-app/src/app/posts/state/post.effects.ts:4]: I also added for feature effects in eager modules.

**Pull request merged**